ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOSX.yml

```diff
@@ -21,6 +21,14 @@
     path: /usr/local/bin/brew
   register: brew
 
+- name: Fix broken fiddle install (Apple Silicone)
+  become: yes
+  become_user: "{{ ansible_user }}"
+  script: scripts/install-fiddle.sh
```

Comment:
https://docs.ansible.com/ansible/latest/modules/gem_module.html?

Reply:
I tried using the gem_module but it didn't work. I think it's because it needed to use the master branch rather than a tag.